sechub-pds-examples/gosec/docker/loadbalancer/nginx.conf

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: MIT
+
+# Based on config from: https://stackoverflow.com/a/54388725
+
+worker_processes  1;
+
+events {
+  worker_connections  1024;
+}
+
+http {
+  access_log /dev/stdout;
+  error_log /dev/stderr;
+
+  server {
+    listen 8444 ssl;
+    server_name localhost;
+
+    ssl_certificate /certificates/localhost.cert;
+    ssl_certificate_key /certificates/localhost.key;
+
+    ssl_protocols TLSv1.2 TLSv1.3;
+    ssl_prefer_server_ciphers on;
+    ssl_dhparam /certificates/certsdhparam.pem;
+    ssl_ecdh_curve secp384r1;
```

Comment:
a known good ssl setup is this:
```
        ssl_protocols TLSv1.2 TLSv1.3;
        ssl_prefer_server_ciphers on;
        ssl_ciphers  HIGH:!aNULL:!MD5:!RC4;
        ssl_session_timeout 1d;
        ssl_session_cache shared:SSL:50m;
        ssl_stapling on;
        ssl_stapling_verify on;
        add_header Strict-Transport-Security max-age=15768000;
```
But I didn't compare with your settings; maybe they overlap.

Reply:
I added some of them. However, `ssl_stapling` does not make sense for a self-signed certificate.